src/unix/linux_like/linux/mod.rs

```diff
@@ -2304,6 +2304,14 @@ pub const ALG_SET_AEAD_AUTHSIZE: ::c_int = 5;
 pub const ALG_OP_DECRYPT: ::c_int = 0;
 pub const ALG_OP_ENCRYPT: ::c_int = 1;
 
+// include/uapi/linux/mman.h
+pub const MAP_SHARED_VALIDATE: ::c_int = 0x3;
+
+// include/uapi/asm-generic/mman-common.h
+#[cfg(not(any(target_arch = "mips", target_arch = "mips64")))]
+pub const MAP_SYNC : ::c_int = 0x080000;
```

Comment:
@JohnTitor It looks like C code generation was failing on mips, mips64. I'm not sure if MAP_SYNC is used or defined for mips. Is this an acceptable way of turning it off?

Reply:
I'll take a look but it should be fine if it isn't defined.

Reply:
I investigated a bit, perhaps we need CI tweaks for mips64. I'll investigate more but it's out of scope here. I'll accept it at this point and follow-up later.